### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by jlebon**
Minor tweaks as I was reading the codebase. See individual commit messages.

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -143,11 +143,11 @@ func (dn *Daemon) triggerUpdate() error {
...
-// validate confirms that the node is actually in the state that it wants to be
-// in. it does this by looking at the elements in the target config and checks
-// if all are present on the node. if any file/unit is missing or there is a
-// mismatch, it re-triggers the update.
-func (dn *Daemon) validate() (bool, error) {
+// currentMatchesDesired confirms that the node is actually in the state that it
+// wants to be in. It does this by looking at the elements in the target config
+// and checks if all are present on the node. Returns true iif there are no
+// mismatches (e.g. files, units... XXX: but not yet OS version).
+func (dn *Daemon) currentMatchesDesired() (bool, error) {
```
Comment (sdemos) [inline pkg/daemon/daemon.go]: as far as bikeshedding this function name goes, I'm partial to keeping it `validate`, but if you think that's not descriptive enough, I think I would go with something like `isDesiredMachineState` or something like that. `currentMatchesDesired` says "helper function for checking if the current annotation equals the desired annotation" to me.

on that note, I notice now that the validate function actually does exactly that, and skips the rest of the validation if they are equal, but I really don't think it should. I think we should be validating the machine state every time we start up, whether or not the annotations imply that we've successfully validated it in the past. we can deal with that later though.

File: pkg/daemon/daemon.go [hunk 2]
```diff
@@ -143,11 +143,11 @@ func (dn *Daemon) triggerUpdate() error {
...
 }
 
-// validate confirms that the node is actually in the state that it wants to be
-// in. it does this by looking at the elements in the target config and checks
-// if all are present on the node. if any file/unit is missing or there is a
-// mismatch, it re-triggers the update.
-func (dn *Daemon) validate() (bool, error) {
+// currentMatchesDesired confirms that the node is actually in the state that it
+// wants to be in. It does this by looking at the elements in the target config
+// and checks if all are present on the node. Returns true iif there are no
```
Comment (sdemos) [inline pkg/daemon/daemon.go]: nit: `s/iif/iff`? if you meant the shorthand for `if and only if`

File: pkg/daemon/daemon.go [hunk 3]
```diff
@@ -107,7 +107,7 @@ func (dn *Daemon) syncOnce() error {
 		return err
 	}
 
-	// watch the annotations.
+	glog.V(2).Infof("Watching for node annotation updates")
```
Comment (ashcrow) [inline pkg/daemon/daemon.go]: :+1:

File: pkg/daemon/daemon.go [hunk 4]
```diff
@@ -73,8 +73,8 @@ func New(
 // function shouldn't return, and should just reboot the node, unless an error
 // occurs, in which case it will return the error up the call stack.
 func (dn *Daemon) Run(stop <-chan struct{}) error {
-	glog.Info("Starting MachineConfigDameon")
-	defer glog.Info("Shutting down MachineConfigDameon")
+	glog.Info("Starting MachineConfigDaemon")
+	defer glog.Info("Shutting down MachineConfigDaemon")
```
Comment (ashcrow) [inline pkg/daemon/daemon.go]: :+1:

File: pkg/daemon/daemon.go (hunk 1 above)
Comment (ashcrow) [reply pkg/daemon/daemon.go]: I'll defer to @sdemos and @jlebon. I'm OK with either.

**Comment by jlebon** [comment]
Updated for comments!

**Pull request merged**